components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -301,20 +301,31 @@ class BrowserToolbar @JvmOverloads constructor(
      * @param contentDescription The content description to use.
      * @param visible Lambda that returns true or false to indicate whether this button should be shown.
      * @param background A custom (stateful) background drawable resource to be used.
+     * @param paddingStart Padding start in PX.
+     * @param paddingTop Padding top in PX.
+     * @param paddingEnd Padding end in PX.
+     * @param paddingBottom Padding end in PX.
      * @param listener Callback that will be invoked whenever the button is pressed
      */
     open class Button(
         imageResource: Int,
         contentDescription: String,
         visible: () -> Boolean = { true },
         @DrawableRes background: Int? = null,
+        private val paddingStart: Int = ACTION_PADDING_DP,
+        private val paddingTop: Int = ACTION_PADDING_DP,
+        private val paddingEnd: Int = ACTION_PADDING_DP,
+        private val paddingBottom: Int = ACTION_PADDING_DP,
```

Comment:
I wonder if we should just go with one padding for all of them for now. It's good enough for our default padding and for removing padding completely.

Reply:
Yes, we can. I just wanted to be as flexible as possible, for the consumers. Should I change it to just one value?

Reply:
What about having overloaded constructors? One with the 4 parameters and another that takes just one add the same value for start, top, end, and bottom?

Reply:
> Yes, we can. I just wanted to be as flexible as possible, for the consumers. 

Yeah, this is more flexible. But it's also more noisy by adding 4 more parameters. I wonder if a `Padding` data class in the base component would be helpful?

Reply:
Let's add the class `Padding`, I was thinking the same!